iou_early_stopping: 0.9
optim:
    max_iter: 30
    losses:
        silhouette_loss:
            active: true
        plane_collision_loss:
            active: true
resolution: 480
mask_grabcut_refinement : true